schema.graphql

```diff
@@ -131,5 +131,24 @@ type ReferenceAnnotation implements Annotation {
   number: Float @search
   tag: String @search
   type: ReferenceAnnotationType!
+  semanticDomains: [SemanticDomain] @hasInverse(field: referenceAnnotations)
+}
+
+type SemanticDomain {
+  id: ID!
+  cid: String!
+  name: String! @search
+  question: String
+  words: [Word!] @hasInverse(field: semanticDomains)
+  parentSemanticDomain: SemanticDomain
+  childSemanticDomains: [SemanticDomain] @hasInverse(field: parentSemanticDomain)
+  referenceAnnotations: [ReferenceAnnotation] @hasInverse(field: semanticDomains)
+}
+
+type Word {
+  id: ID!
+  text: String! @search
+  semanticDomains: [SemanticDomain!]
+  isoLanguage: ISOLanguage!
 }
```

Comment:
I have added the words to the schema to simplify the reproducibility of the semantic domain tagging. We could also remove them from the schema. What do you prefer, @dwhitena?